publish-extensions.js

```diff
@@ -78,19 +75,59 @@ const readFile = util.promisify(fs.readFile);
         console.log(error);
       }
 
-      // Publish the extension.
-      const options = { packagePath: path.join('/tmp/repository', extension.location || '.') };
-      if (yarn) {
-        options.yarn = true;
+      if (extension.download) {
+        if (extension.repository) {
+          console.warn('[WARN] Ignoring `repository` property because `download` was given.')
+        }
+        if (extension.checkout) {
+          console.warn('[WARN] Ignoring `checkout` property because `download` was given.')
+        }
+        if (extension.prepublish) {
+          console.warn('[WARN] Ignoring `prepublish` property because `download` was given.')
+        }
+        if (extension.location) {
+          console.warn('[WARN] Ignoring `location` property because `download` was given.')
+        }
+
+        // Download the extension package from a GitHub release
```

Comment:
The `.vsix` can actually be downloaded from anywhere.

```suggestion
        // Download the extension package, e.g. from a GitHub release
```